doc/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docs/base:latest
+MAINTAINER Sven Dowideit <SvenDowideit@docker.com> (@SvenDowideit)
+
+# to get the git info for this repo
+COPY . /src
+
+# Reset the /docs dir so we can replace the theme meta with the new repo's git info
+RUN git reset --hard
+
+RUN grep 'var Version' /src/version/version.go  | sed 's/.*"\(.*\)".*/\1/' > /docs/VERSION
```

Comment:
Using `git describe` here might be more reliable. This file contains only a rough guess at the version. The real value is set through the linker.